helpers.py

```diff
@@ -2,10 +2,38 @@
 from math import log, sin
 
 from selenium import webdriver
+from selenium.common import TimeoutException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.ui import WebDriverWait
 
 base_url = "https://suninjuly.github.io"
 
 
+def assert_if_user_is_authorized(browser):
+    """Assert function to verify user authorization.
+    Raises an assertion error if the user is not authorized.
+    """
+    assert check_if_user_is_authorized(
+        browser
+    ), "User is guest, the answer can't be sent"
+
+
+def check_if_user_is_authorized(browser):
```

Comment:
Тут точно есть смысл делать отдельную функцию для этого, а не выполнять всё в `assert_if_user_is_authorized`? Если да, то...

```suggestion
def _check_if_user_is_authorized(browser):
```

Reply:
И создай, пожалуйста, Issue насчёт обновления проверки авторизации в соседнем тесте и назначь меня исполнителем 😉

Reply:
итого отдельную функцию убрала, добавила проверку на закрытие попапа в assert function

Reply:
сделала задачу, #44